src/objectTranslations/Account-de.objectTranslation

```diff
@@ -149,4 +149,20 @@
         <label>Schenkungen gesamt</label>
         <name>TotalOppAmount__c</name>
     </fields>
-    </CustomObjectTranslation>
+    <fields>
```

Comment:
@ptewson-sfdo Looks like this is pulling in translations from the one package that it is dependent on. Can you add the following xpath to the cci task and re-run the cci task? That should strip these out.
```
                - path: src/*ranslations/**/*ranslation
                  xpath: /*/*[re:match(ns:name, '...+__..+')]
```